cluster/manifests/kube-cluster-autoscaler/daemonset.yaml

```diff
@@ -41,6 +41,7 @@ spec:
           - --skip-nodes-with-system-pods=false
           - --skip-nodes-with-local-storage=false
           - --scale-up-cloud-provider-template=true
+          - --expander=prefer-spot
```

Comment:
This flag works in both versions? (You only updated one of them).

Reply:
Oops, you're right.